week-1-practise/level-1/10-function-output-3.js

```diff
@@ -1,6 +1,6 @@
 function concatenate(firstWord, secondWord, thirdWord) {
-    // Write the body of this function to concatenate three words together
+  return firstWord.concat(secondWord, thirdWord);
```

Comment:
Good to use the fact that concat takes multiple parameters!

Reply:
Thanks and sorry for delay in responding. Actually I thought we will be in touch just through the slack. I didn't notice your comment on my GitHub.